spec/services/comment_service_spec.rb

```diff
@@ -22,6 +61,15 @@
       end
     end
 
+    context "bucket author is subcribed to personal activity, but is also commenter" do
```

Comment:
this is already tested up on line#49  - don't bother taking it out though. Even reading this is costing too much!